packages/itwinui-css/src/tabs/default.scss

```diff
@@ -78,7 +77,7 @@
 
   &.iui-large {
     .iui-tab {
-      height: $iui-component-height-large;
+      min-height: $iui-component-height-large;
```

Comment:
How would you feel about setting a `max-height` or a `-webkit-line-clamp` to prevent extreme cases like this?
![Screen Shot 2022-06-09 at 12 46 21 PM](https://user-images.githubusercontent.com/849817/172900668-0f444901-c810-464e-b64d-cfa0ee58fa21.png)